### PR TITLE
Handles Jpeg Image Files

### PR DESCRIPTION
**Pull request by gilroman**

### CODE REVIEW

File: src/main/java/gil/server/controllers/PersonController.java [hunk 1]
```diff
@@ -35,11 +35,12 @@ public PersonController(JSONDataStore dataStore) {
                     Integer pid = person.getId();
                     String name = person.getName();
                     String email = person.getEmail();
+                    String body = "{ \"id\": " + pid + ", \"name\": \"" + name + "\", \"email\": \"" + email + "\"}";
```
Comment (ssunday) [inline src/main/java/gil/server/controllers/PersonController.java]: This is basically, json, right? Can you use a json builder so it isn't string concatenation?

File: src/main/java/gil/server/controllers/StaticFileHandler.java [hunk 2]
```diff
@@ -9,14 +9,23 @@
...
 
                 try {
-                    String body = StaticFileUtils.getFileContent(uri);
+                    switch(StaticFileUtils.getFileType(uri)) {
+                        case "TEXT":
+                            body = StaticFileUtils.getTextFileContent(uri);
+                            response.addHeader(HTTPProtocol.CONTENT_TYPE,"text/html; charset=UTF-8");
+                            break;
+                        case "JPEG":
+                            body = StaticFileUtils.getImageFileContent(uri);
```
Comment (ssunday) [inline src/main/java/gil/server/controllers/StaticFileHandler.java:22]: if the body is being set in the switch, why not make the header content type var a variable too and set the response with it below with everyone else

File: src/main/java/gil/server/controllers/PersonController.java (hunk 1 above)
Comment (gilroman) [reply src/main/java/gil/server/controllers/PersonController.java]: Great suggestion. Thank you!

File: src/main/java/gil/server/controllers/StaticFileHandler.java (hunk 2 above)
Comment (gilroman) [reply src/main/java/gil/server/controllers/StaticFileHandler.java:22]: Very nice. I've refactored the handler to implement this.

**Comment by gilroman** [comment]
Merged to master.

**Pull request closed without merging**